src/fisher.jl

```diff
@@ -24,6 +24,33 @@
 
 export FisherExactTest
 
+"""
+    FisherExactTest(a::Integer, b::Integer, c::Integer, d::Integer)
+
+Perform Fisher's exact test of the null hypothesis that the success probabilities ``a/c``
+and ``b/d`` are equal, that is the odds ratio ``(a/c) / (b/d)`` is one, against the
+alternative hypothesis that they are not equal.
+
+The contingency table is structured as:
+
+| -  | X1 | X2 |
+|:--:|:--:|:--:|
+|*Y1*| a  | b  |
+|*Y2*| c  | d  |
+
+!!! note
+    The [`Base.show`](@ref) output contains the conditional maximum likelihood estimate of the odds ratio
```

Comment:
When you build the docs, what does this end up linking to?

Reply:
Unfortunately it seems it's not possible at the moment: https://github.com/JuliaDocs/Documenter.jl/issues/425. Let's go without links for now.